changedetectionio/__init__.py

```diff
@@ -744,6 +744,27 @@ def preview_page(uuid):
                                  uuid=uuid)
         return output
 
+    @app.route("/api/<string:uuid>/current_snapshot", methods=['GET'])
```

Comment:
I would like it to be

`/api/<string:uuid>/snapshot/current`

it feels more future-proof that way, maybe someone want like "first" snapshot or some date/time, or some snapshot by key in the future

Reply:
Sure, totally makes sense